src/lib/handlers.js

```diff
@@ -37,27 +33,42 @@ import resetUserPassword from './auth/reset-user-password';
 const domain = process.env.DOMAIN;
 const mailgun = require('mailgun-js')({ apiKey: process.env.MAILGUN_API_KEY, domain });
 
-export function postEventHandler (req, res, next) { // eslint-disable-line no-unused-vars
+
+export function postEventHandler (req, res, next) {
   const event = req.body.event;
   if (!event) {
     return res.status(422).send({ error: 'Missing event data' });
   }
-  const data = Object.assign(event, { host_user_id: req.user.user_id });
+  const data = { ...event, host_user_id: req.user.user_id };
   const code = shortid.generate();
   data.code = code;
   saveEvent(client, data)
-    .then(() => {
-      res.json({ code });
+    .then((event_id) => {
+      req.subject_user_id = req.user.user_id;
+      req.event_id = event_id;
+      req.informAllInvitees = false;
+      req.responseStatusCode = 201;
+      req.responseData = { code };
+      next(); // --> updateFeeds
     })
     .catch((err) => {
       return res.status(500).send({ error: err });
     });
 }
 
+export function prepareToDeleteEvent (req, res, next) {
+
+  req.subject_user_id = req.user.user_id;
+  req.event_id = req.params.event_id;
+  req.informAllInvitees = true;
+  next(); // --> updateFeeds
+}
+
 export function deleteEventHandler (req, res, next) {
+  // updateFeeds happens before this step
   deleteEvent(client, req.params.event_id)
-  .then((deleted_event_id) => {
-    res.json(deleted_event_id);
+  .then(() => {
+    return res.status(204).end();
```

Comment:
@heron2014 yeah.  there's no data to send back, so just a matter of ending the request.  The client will receive a 204 and that will be the end